src/compaction.c

```diff
@@ -29,6 +30,7 @@ typedef struct AvgContext
 {
     double val;
     double cnt;
+    u_int64_t overflow;
```

Comment:
```suggestion
    bool overflow;
```

Reply:
@danni-m But Redis doesn't supply a serialization function for bool

Reply:
@danni-m Can I use char in here instead of bool like the isResetted flag?